src/scripts/main.js

```diff
@@ -1,3 +1,21 @@
 'use strict';
 
-// write code here
+const list = document.querySelector('ul');
+
+list.querySelectorAll('li').forEach((el) => {
+  const span = document.createElement('span');
+
+  span.textContent = el.firstChild.textContent.trim();
```

Comment:
Ensure that `el.firstChild` is not null before calling `textContent.trim()`. This will avoid runtime errors if the list item is empty or does not have a text node as its first child.